src/components/page/page_header/page_header_content.tsx

```diff
@@ -30,7 +30,7 @@ type Tab = EuiTabProps & {
   label: ReactNode;
 };
 
-export type EuiPageHeaderContentTitle = {
+export interface EuiPageHeaderContentTitle {
```

Comment:
`type` to `interface` refactor isn't strictly necessary, but sometimes using `interface extends` results in cleaner output. When possible, I like to use `interface` over `type`

Reply:
👍  TIL